lib/mail/fields/content_type_field.rb

```diff
@@ -144,10 +144,9 @@ def sanatize( val )
 
       # TODO: check if there are cases where whitespace is not a separator
       val = val.
+        gsub(/\s[\w\-]+:\s.*\\r\\n$/, ''). # cases with botched fields like Content-Type: text/html Content-Transfer-Encoding: 8bit\r\n
```

Comment:
Sure, I do understand you want to strike the right balance for your vision of the library's future.

All I am saying is that throwing an Exception is not an answer to my quest. If an Exception is raised, the caller simply has no option but to fail the processing completely. The caller cannot fixup-and-proceed-from-where-it-failed, because there is no way of knowing where in the email body processing has failed and in what way. So I cannot fix this problem by acrobatics in my code or by fixing up the email body.

I don't see a problem that you won't merge my pull request. I am not trying to persuade you to change your acceptance criteria.

If you see some emails from your regression suite for which the change fails, it'd be great to see those emails, too.